src/Features/CSharp/Portable/InlineHints/CSharpInlineTypeHintsService.cs

```diff
@@ -80,6 +80,15 @@ public CSharpInlineTypeHintsService()
                 }
             }
 
+            if (node is ImplicitObjectCreationExpressionSyntax { Parent: ArgumentSyntax } implicitNew)
```

Comment:
I think this actually needs to go into https://github.com/dotnet/roslyn/blob/master/src/Features/CSharp/Portable/InlineHints/CSharpInlineParameterNameHintsService.cs, right @CyrusNajmabadi?